<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryWindow</class>
 <widget class="QMainWindow" name="QueryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>601</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ContextMenuPolicy::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Let's run a query on the database</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="text">
      <property name="contextMenuPolicy">
       <enum>Qt::ContextMenuPolicy::ActionsContextMenu</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStartTransaction"/>
   <addaction name="separator"/>
   <addaction name="actionCommit"/>
   <addaction name="actionRollback"/>
   <addaction name="separator"/>
   <addaction name="actionExecute"/>
  </widget>
  <action name="actionStartTransaction">
   <property name="text">
    <string>Start &amp;transaction</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+T</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionCommit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Commit</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRollback">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Roll&amp;back</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+R</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionExecute">
   <property name="text">
    <string>E&amp;xecute</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Return</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Shift+Esc</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
